TypeCobol/Compiler/Sql/CodeElements/SqlCodeElementBuilder.cs

```diff
@@ -303,5 +303,12 @@ public LockTableStatement CreateLockTableStatement(CodeElementsParser.LockTableS
 
             return new LockTableStatement(tableName, partitionId, mode);
         }
+
+        public ReleaseSavepointStatement CreateReleaseSavepointStatement(CodeElementsParser.ReleaseSavepointStatementContext context)
+        {
+            if (context.savepoint_name == null) return null;
```

Comment:
No. Like other statements, the builder should create them even if they are invalid/incomplete.